insee/insee.ml

```diff
@@ -0,0 +1,168 @@
+(* Copyright (c) 2019 Ludovic LEDIEU *)
+(* Adjusted for gwrepl by Henri *)
+(*
+  use: cat <script.ml> | [ GWREPL_VERBOSE=1 ] [ GWREPL_FORCE_UNPACK=1 ]
+   [ GWREPL_NOPROMPT=1 ] gwrepl.exe [script_arg1] ...
+
+*)
+#mod_use "/Users/Henri/GitHub/hgouraud/geneweb/lib/date.ml" ;;
+#mod_use "/Users/Henri/GitHub/hgouraud/geneweb/lib/lock.ml" ;;
+#mod_use "/Users/Henri/GitHub/hgouraud/geneweb/lib/config.ml" ;;
+#mod_use "/Users/Henri/GitHub/hgouraud/geneweb/lib/output.ml" ;;
+#mod_use "/Users/Henri/GitHub/hgouraud/geneweb/lib/gwlib.ml" ;;
+#mod_use "/Users/Henri/GitHub/hgouraud/geneweb/lib/GWPARAM.ml" ;;
+
+let open Def in
+let open Gwdb in
+let open Arg in
```

Comment:
Je suppose unused

Reply:
sans le `let open Gwdb in`, j'ai
`Error: Unbound value get_birth` !!
Les deux autres semblent effectivement inutiles.

Reply:
Je parlais uniquement du `Arg`, mais j'ai pas vérifié.